source/types.ts

```diff
@@ -202,10 +203,44 @@ export interface NormalizedOptions extends Options {
 	cookieJar?: PromiseCookieJar;
 	maxRedirects: number;
 
+	// Other values
+	decompress: boolean;
+	isStream: boolean;
+	throwHttpErrors: boolean;
+	ignoreInvalidCookies: boolean;
+	cache: CacheableRequest.StorageAdapter | false;
+	responseType: ResponseType;
+	resolveBodyOnly: boolean;
+	followRedirect: boolean;
+	useElectronNet: boolean;
+	methodRewriting: boolean;
+	context: {[key: string]: any};
+
 	// UNIX socket support
 	path?: string;
 }
 
+/*
```

Comment:
Cannot decide which is better. Both solutions work. Should be `NormalizedOptions` inherited from `Options` or `DefaultOptions`?

Reply:
Semantically, it makes more sense for it to inherit from `Options`, but I might be missing some context. Are there any downsides/upsides to either way?

Reply:
It's just that `DefaultOptions` are already normalized but some properties are **excluded** there. 

I agree. Let's go with `Options`.